apply-roles

```diff
@@ -122,6 +128,21 @@ generate_custom_role_definition() {
     }"
 }
 
+generate_tenant_custom_role_definition() {
+  local tenant_id="${1}"
+
+  echo "{ \
+      \"Name\": \"${TENANT_CUSTOM_ROLE_NAME}\", \
+      \"Description\": \"Tenant level custom role for Cloud Conformity access.\", \
```

Comment:
🤔 This shouldn't reference `Cloud Conformity` it should probably be `Cloud One Conformity`

```suggestion
      \"Description\": \"Tenant level custom role for Cloud One Conformity access.\", \
```

Reply:
Updated!